src/pyhf/infer/calculators.py

```diff
@@ -120,7 +120,7 @@ def cdf(self, value):
             >>> import pyhf
             >>> pyhf.set_backend("numpy")
             >>> bkg_dist = pyhf.infer.calculators.AsymptoticTestStatDistribution(0.0)
-            >>> bkg_dist.cdf(0.0)
+            >>> print(bkg_dist.cdf(0.0))
```

Comment:
Did something change RE: the way doctest works where the `print`s are now necessary? We had always treated this like REPL behavior, not script behavior.

Reply:
No, numpy changed. ubuntu-latest has numpy 2.x which gives `np.float64(0.0)` while macos-13 uses numpy 1.x which gives `0.0` -- we can't write a test that relies on `repr` for two diff major versions of numpy.

Reply:
```python
>>> import numpy as np
>>> np.__version__
'2.0.2'
>>> np.float64(0.0)
np.float64(0.0)
>>> print(np.float64(0.0))
0.0
```

and

```python
>>> import numpy as np
>>> np.__version__
'1.26.4'
>>> np.float64(0.0)
0.0
>>> print(np.float64(0.0))
0.0
```